### PR TITLE
chore(prettier): add configuration

### PR DESCRIPTION
**Pull request by AbhiGattineni**
Added Prettier for consistent code formatting.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (RushilBasappa) [inline .prettierrc:1]: tabwidth should be 2

File: src/components/index.tsx [hunk 2]
```diff
@@ -1,3 +1,3 @@
-import Button from "./Button/Button";
+import Button from "../components/Button/Button";
```
Comment (RushilBasappa) [inline src/components/index.tsx]: import Button from "./Button/Button";

This is correct, there is no need to go a level back ../components

**Review by RushilBasappa — CHANGES_REQUESTED** [review]

File: tsconfig.json [hunk 3]
```diff
@@ -22,16 +22,15 @@
     // modules
     "baseUrl": "src/",
     "paths": {
-      "Components/*": ["Components/*"],
+      "Components/*": ["components/*"]
```
Comment (RushilBasappa) [inline tsconfig.json]: Why?
wont this break the code

File: src/App.tsx [hunk 4]
```diff
@@ -1,13 +1,15 @@
 import { useState } from "react";
 import "./App.css";
-import { Button } from "Components";
+import { Button } from "./components";
```
Comment (RushilBasappa) [inline src/App.tsx]: same here
This is relative path. Need to use absolute path
This change is not needed

File: index.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```
Comment (RushilBasappa) [inline index.html:1]: check why did this change to small case

**Review by RushilBasappa — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 4 above)
Comment (AbhiGattineni) [reply src/App.tsx]: If I don't make these changes , code is showing errors in my IDE.

File: tsconfig.json (hunk 3 above)
Comment (AbhiGattineni) [reply tsconfig.json]: If I don't make these changes , code is showing errors in my IDE.

**Review by RushilBasappa — APPROVED** [review]

**Pull request merged**